src/language/parser.js

```diff
@@ -63,16 +58,28 @@ import {
   OBJECT_FIELD,
 
   DIRECTIVE,
-
-  NAMED_TYPE,
-  LIST_TYPE,
-  NON_NULL_TYPE,
 } from './kinds';
 
+import {
+  makeParser,
+  peek,
+  skip,
+  loc,
+  any,
+  many,
+  parseName,
+  expect,
+  parseType,
+  parseNamedType,
+  unexpected,
+  expectKeyword,
+  advance,
+} from './parserCore';
```

Comment:
The role of this is a little muddy. I wonder if there's a better place for `parseName` and the `parseType` group to exist? They're definitely specific parser rules not parser core functionality.

For the short term, and for the sake of maintaining the clarity of this document relative to the spec, I would suggest (deep breath) copying these few functions directly in `schema/parser.js`. I think DRY is working against us here.